sass/materialize.scss

```diff
@@ -1,17 +1,16 @@
 @charset "UTF-8";
 
-// Mixins
-// @import "components/prefixer";
-@import "components/mixins";
-@import "components/color";
-
-// Variables;
+// 1. Variables
 @import "components/variables";
 
-// Reset
+// 2. Reset
 @import "components/normalize";
 
-// components
+// 3. Mixins
+@import "components/prefixer";
```

Comment:
Haha I thought you might catch that! If you take a look that file though, you'll see that nearly everything inside it is commented out except for a single mixin which isn't used anywhere. I figured it would be harmless to uncomment it for now so save a future headache where someone puts things in that file and wonders why they're not working ;)